lib/yookassa/payments.rb

```diff
@@ -31,5 +36,13 @@ def cancel(payment_id:, idempotency_key: SecureRandom.hex(10))
     private
 
     attr_reader :api
+
+    def validate_payment(data)
+      contract = Yookassa::Contracts::PaymentContract.new
+      result = contract.call(data)
+      return if result.success?
+
+      raise CreatePaymentError, result.errors.to_h
```

Comment:
Our api call can return not only success, but also errors. That would be good to unify the returning value (search for error entity in the project)